UITests/Campaigns/modal-controls.json

```diff
@@ -0,0 +1,88 @@
+{
+    "campaignId": "modal-controls",
+    "maxImpressions": 999,
+    "type": 1,
+    "isTest": false,
+    "triggers":
+    [
+        {
+            "type": 1,
+            "eventType": 4,
+            "eventName": "second activity",
+            "attributes": [
+            {
+                "name": "click",
+                "value": "true",
+                "type": 4,
+                "operator": 1
+            }]
+        }
+    ],
+    "messagePayload":
+    {
+        "title": "test",
+        "messageBody": "test",
+        "header": "Test Campaign",
+        "titleColor": "#000000",
+        "headerColor": "#000000",
+        "messageBodyColor": "#000000",
+        "backgroundColor": "#ffffff",
+        "frameColor": "#ffffff",
+        "resource":
+        {
+            "assetsUrl": "",
+            "imageUrl": "file:///System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/ClippingPicture.icns",
```

Comment:
Think this file will be around in future versions of macOS? 🤔

Reply:
Yeah, might not be. On real device it'll probably fail but are we ever going to run those tests on a real device?
I didn't want to put any http url based image. I can try to embed some image in the test bundle but the path will be different on real device as well. Do you have any better idea?

Reply:
Can it use an image embedded as a bundle resource?

Reply:
Bump

Reply:
Probably. If putting image name as a url is going to work...
But yeah, it is a better idea

Reply:
There's no way to get a static url to a resource :( I used Shock url mocking instead